Assets/Scripts/BuddyManager.cs

```diff
@@ -23,6 +23,6 @@ public static void DecrementAllBuddyResources()
 			buddyStat.DecrementResources();
 		}
 
-		instance._buddyStats.RemoveAll( buddyStat => buddyStat.isAlive == false ); // Predicate statement
+		instance._buddyStats.RemoveAll( buddyStat => buddyStat.isAlive == false );
```

Comment:
I'd recommend keeping this comment in here because I'm not going to remember what that => symbol means the next time I look at this and I won't even know what it's called.